cvat/apps/dataset_manager/default_settings.py

```diff
@@ -6,9 +6,3 @@
 
 DATASET_CACHE_TTL = int(os.getenv("CVAT_DATASET_CACHE_TTL", 60 * 60 * 24))
```

Comment:
Why this variable is left here? Maybe it's better to implement the same logic for backups as well? Either way, it doesn't seem to be the topic of this PR.

Reply:
If I remember correctly, backup logic also used this setting